code/go/0chain.net/smartcontract/storagesc/allocation.go

```diff
@@ -484,6 +497,10 @@ func (uar *updateAllocationRequest) validate(
 		}
 	}
 
+	if uar.FileOptions > 63 || uar.FileOptions < 0 {
```

Comment:
what if fileOptions is 10? is it valid?

Reply:
Yes, then in binary it will be `1010` which would mean third parties are allowed to move/delete files on this allocation. 

https://github.com/0chain/0chain/pull/1453/files/54bbb5896722d3afc510e384331a149445004d22#diff-9daedcedaa17e8d43dce32bac14380fececa220fadb3ae99d55cd0b46cfd0cf6R655-R664

Reply:
Okay it's a multi operation. Cool